From 44b878afd7bec90538b04a4d90894a4279ab4835 Mon Sep 17 00:00:00 2001
From: ptrcnull <git@ptrcnull.me>
Date: Fri, 17 Nov 2023 19:19:19 +0100
Subject: [PATCH] change default opt level to 0

that's also what python-installer uses, and it would
make `gpep517 install-wheel` invokations more neat
---
 gpep517/__main__.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/gpep517/__main__.py b/gpep517/__main__.py
index d519c75..2dc0b49 100644
--- a/gpep517/__main__.py
+++ b/gpep517/__main__.py
@@ -350,9 +350,9 @@ def add_install_args(parser):
                        f"(default: {sys.executable})")
     group.add_argument("--optimize",
                        type=parse_optimize_arg,
-                       default=[],
+                       default=[0],
                        help="Comma-separated list of optimization levels "
-                       "to compile bytecode for (default: none), pass 'all' "
+                       "to compile bytecode for (default: 0), 0s 'all' "
                        "to enable all known optimization levels (currently: "
                        f"{', '.join(str(x) for x in ALL_OPT_LEVELS)})")
 
-- 
2.44.0

